elastic/kibana
# [Infra] Add the logs tab to container views

Closes #184770 

## Summary

This PR adds the logs tab to container views.

### Testing

- Enable Container view from Infra settings
- Go to Inventory and select Docker Containers from the "Show" drop-down
- Click on a container
   - The logs tab should be visible 
- Click on the Logs tab
- The search functionality and the link (with the filter) should work

![image](https://github.com/elastic/kibana/assets/14139027/c0720a47-50e5-4244-9f50-53882fc5ed53)

- Click on "Open as Page"
   - The logs tab should be visible 
- The search functionality and the link (with the filter) should work

<img width="1717" alt="image" src="https://github.com/elastic/kibana/assets/14139027/90161dd7-ae66-4274-bc99-39e4c9233fa7">


https://github.com/elastic/kibana/assets/14139027/0ce5761c-cc39-4d55-87ba-a6c12cc7315c

- ⚠️  If some of the containers on the edge cluster don't have logs in the logs settings the `container.id` can be added and used in inventory to find a container with logs ( This should be added for the docker containers on edge soon - for now it can be checked for a non-docker container )

![image](https://github.com/elastic/kibana/assets/14139027/90d7f0e4-384d-4cc8-aea0-6ff10713450d)

Example with a container id (with logs)

![image](https://github.com/elastic/kibana/assets/14139027/162f37ce-8ba8-4c8d-835d-f2d23c3bc0ec)
